### ISSUE TITLE
iOS cocoapods install error and compile error due to unknown type

### ISSUE THREAD

**Issue by mthongvanh** [sebkoller/certificate_pinning_httpclient#2]
on flutter 3.0.5, i encountered two issues while building for iOS. when running `pod install` i received the following error:
```
    -> Fetching podspec for `certificate_pinning_httpclient` from `.symlinks/plugins/certificate_pinning_httpclient/ios`
    [!] The `certificate_pinning_httpclient` pod failed to validate due to 2 errors:
        - ERROR | attributes: Missing required attribute `homepage`.
        - ERROR | attributes: Missing required attribute `source`.
```
and after fixing that error, i hit another error when trying to build:
```
    ../../../.pub-cache/hosted/pub.dartlang.org/certificate_pinning_httpclient-0.0.1/lib/certificate_pinning_httpclient.dart:18:33: Error: Type 'Uint8List' not found.
      static final Map<String, List<Uint8List>?> _hostCertificates =
                                    ^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/certificate_pinning_httpclient-0.0.1/lib/certificate_pinning_httpclient.dart:27:22: Error: Type 'Uint8List' not found.
      static Future<List<Uint8List>?> _getHostCertificates(Uri url) async {
                         ^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/certificate_pinning_httpclient-0.0.1/lib/certificate_pinning_httpclient.dart:57:22: Error: Type 'Uint8List' not found.
      static Future<List<Uint8List>> _hostPinCertificates(
                         ^^^^^^^^^
```

output from `flutter doctor -v`
```
[✓] Flutter (Channel unknown, 3.0.5, on macOS 12.6 21G115 darwin-x64, locale en-US)
    • Flutter version 3.0.5 at /Users/michael/fvm/versions/3.0.5
    • Upstream repository unknown
    • Framework revision f1875d570e (5 months ago), 2022-07-13 11:24:16 -0700
    • Engine revision e85ea0e79c
    • Dart version 2.17.6
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/michael/Library/Android/sdk
    • Platform android-33, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 14.0)
    • Xcode at /Applications/Xcode1400.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.12+0-b1504.28-7817840)

[✓] VS Code (version 1.73.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.54.0

[✓] Connected device (4 available)
    • iPhone 13 (mobile) • B5D33A2C-9768-466D-B184-E6810CF62B81 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-15-0 (simulator)
    • macOS (desktop)    • macos                                • darwin-x64     • macOS 12.6 21G115 darwin-x64
    • Chrome (web)       • chrome                               • web-javascript • Google Chrome 108.0.5359.98

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```
[https://github.com/sebkoller/certificate_pinning_httpclient/pull/1](https://github.com/sebkoller/certificate_pinning_httpclient/pull/1) pull request submitted to fix these two issues

**Comment by sebkoller** [comment]
Thank you @mthongvanh for the pull request!
I actually never tested iOS builds (I know, I know ...)

I just released `0.0.2` with your fix.

**Comment by mthongvanh** [comment]
i'm glad to help @sebkoller and an even bigger thank you for putting this package together
